### PR TITLE
feature: created a new index.html file

### PR DESCRIPTION
**Pull request by YHussei**
This code needs reviewing! I created a new webpage

### CODE REVIEW

File: index-hussein.html
Comment (JT-code-dev) [inline index-hussein.html:1]: Great Job!! Did you find my file? feature-thompson?

**Review by JT-code-dev — APPROVED** [review]
ok I think its all set to be merged. great explanation.

File: index-hussein.html
Comment (YHussei) [reply index-hussein.html:1]: thank you! very new to this, how do I find yours?

Comment (YHussei) [reply index-hussein.html:1]: was looking for your branch, I couldnt seem to find it

might be looking in the wrong place

Comment (YHussei) [reply index-hussein.html:1]: Screenshot 2024-10-29 205454

**Pull request merged**